src/transformState.js

```diff
@@ -62,7 +62,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const obj of transforms) {
+    for (const key in obj) {
+      if (obj[key] === 'addProperties') {
+        Object.assign(state, obj.properties);
+      }
+
+      if (obj[key] === 'clear') {
+        for (const k in state) {
```

Comment:
```suggestion
        for (const field in state) {
```
what does key means?